packages/docusaurus-theme-openapi-docs/src/theme/ApiItem/index.tsx

```diff
@@ -11,14 +11,14 @@ import React from "react";
 
 import BrowserOnly from "@docusaurus/BrowserOnly";
 import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
+import { DocProvider } from "@docusaurus/plugin-content-docs/client";
 import { HtmlClassNameProvider } from "@docusaurus/theme-common";
-import { DocProvider } from "@docusaurus/theme-common/internal";
 import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import useIsBrowser from "@docusaurus/useIsBrowser";
 import { createAuth } from "@theme/ApiExplorer/Authorization/slice";
 import { createPersistanceMiddleware } from "@theme/ApiExplorer/persistanceMiddleware";
-import DocItemLayout from "@theme/ApiItem/Layout";
 import type { Props } from "@theme/DocItem";
+import DocItemLayout from "@theme/DocItem/Layout";
```

Comment:
It looks like this change caused a visual update to generated API pages.  Previously, the pagination at the bottom of the page was constrained to the center portion of the page, now after this update, the pagination also stretches into the right sidebar.  We prefer the way it was previously.  Is there any way to restore that?

Before:
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/4ecd038f-9985-4e4f-a088-52d2d43f568d">

After:
<img width="1510" alt="Arc 2024-08-27 08 48 39" src="https://github.com/user-attachments/assets/f3fd0747-e1e8-49d7-be01-b722b8d0cbac">

Reply:
Thanks!  I caught it because of a visual regression testing tool, have you considered adding something like that to the demo, to catch small changes like this that are potentially easy to miss?

Reply:
I have - would you be willing to share your setup? I know Docusaurus uses Playwright + Argos, but I was hoping for something self-contained that can run entirely in GH Actions. Thanks!

Reply:
I have used Percy in the past and recently switched to using [lost-pixel](https://github.com/lost-pixel/lost-pixel).  I'm using their platform version via a github action (which is free for open source projects), but they also have a purely open-source self-hosted model available that would probably meet your needs from what I can tell.